docs/content/users/install/ddev-installation.md

```diff
@@ -114,9 +114,38 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
     * Installing Chocolatey package manager (optional).
     * One time initialization of mkcert.
     * Installing WSL2 and installing a distro like Ubuntu.
-    * Installing or upgrading to the latest Docker Desktop for Windows with WSL2 enabled.
+    * (Optionally) Installing Docker Desktop for Windows and enabling WSL2 integration with the distro.
     * Installing DDEV inside your distro.
 
+    ### WSL2 with Docker-ce Inside Scripted Installation
```

Comment:
Refer to [Docker CE](https://docker-docs.netlify.app/install/), shorten heading, and use the phrase “install script” to be more consistent with [the rest of this page](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/#system-requirements):

```suggestion
    ### WSL2 + Docker CE Inside Install Script
```

On the fence about the word “inside” here, but I assume it’s critical to point out that Docker CE will live inside the WSL2 distro.